pkg/system/phase1_verifying.go

```diff
@@ -190,22 +192,30 @@ func (r *Reconciler) CheckJoinSecret() error {
 		return util.NewPersistentError("InvalidJoinSecert",
 			"JoinSecret is missing mgmt_addr")
 	}
-	if r.JoinSecret.StringData["bg_addr"] == "" {
-		return util.NewPersistentError("InvalidJoinSecert",
-			"JoinSecret is missing bg_addr")
-	}
-	if r.JoinSecret.StringData["md_addr"] == "" {
-		return util.NewPersistentError("InvalidJoinSecert",
-			"JoinSecret is missing md_addr")
-	}
-	if r.JoinSecret.StringData["hosted_agents_addr"] == "" {
-		return util.NewPersistentError("InvalidJoinSecert",
-			"JoinSecret is missing hosted_agents_addr")
-	}
 	if r.JoinSecret.StringData["auth_token"] == "" {
 		return util.NewPersistentError("InvalidJoinSecert",
 			"JoinSecret is missing auth_token")
 	}
+
+	annotationValue, exists := util.GetAnnotationValue(r.NooBaa.GetAnnotations(), "remote-client-noobaa")
+	annotationBoolVal := true
+	if exists {
+		annotationBoolVal = strings.ToLower(annotationValue) == strconv.FormatBool(true)
```

Comment:
Again, unless I'm missing somthing, I don't see a reason to use `strconv.FormatBool(true)` and not `"true"`